paf-mvp-core-js/src/express/express-apps.ts

```diff
@@ -19,24 +19,28 @@ export class MainApp {
  */
 export class VHostApp {
   constructor(public name: string, public hostName: string, public expressApp: Express = express()) {
-    addMiddlewares(this.expressApp);
+    this.addCookieParser();
+
+    this.addPostBodyParser();
+
+    // Systematically redirect HTTP requests to HTTPs
+    this.ensureHttps();
```

Comment:
yes, to mock it

Reply:
Not clear for me. We create something just for mocking it later?

Reply:
I refactored this file (based on top level functions, to replace it with a class and methods, which I believe is cleaner) to be able to mock it in tests.
I think it's a common pattern when adding unit or integration tests, to refactor the "production" code to make it testable, isn't it?

Reply:
What called to my mind is that I don't see any change in terms of mocking here: for instance, the `expressApp` was mockable before and this change is isometric concerning that.

The only thing that I see is that it is a bit cleaner (and thank you for that). However, I am wondering if it was required for this PR and can be added to another independent PR because this one is already big.

Reply:
here: https://github.com/OneKey-Network/OneKey-implementation/pull/257/files#diff-e872e8864034807bafa5060bf661c1e3c5f947e1a6059efdd3564fb1a84d269cR6

Reply:
Discussed offline. We were aligned on some adjustments (e.g having a parameter). Therefore, I am validating the PR as I believe that your changes will be simple and don't require a review. However, don't hesitate to ping me if you want that I check something.

Reply:
Thanks. Fixed in last commit with a parameter and no more mocking